lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -350,13 +360,33 @@ def asm_reverse_http(opts={})
         call ebp
         test eax,eax
         jnz allocate_memory
+      ^
+
+    if retry_wait
+      asm << %Q^
+        set_wait:
+          push #{retry_wait}     ; dwMilliseconds
+          push 0xE035F044        ; hash( "kernel32.dll", "Sleep" )
```

Comment:
Damn, I didn't realise we still had hard-coded hashes in here. I'll get another PR together that uses the `Rex` function to generate this on the fly (along with all other instances in this file).